src/peft/helpers.py

```diff
@@ -170,23 +178,23 @@ def set_adapter_scale(model, alpha):
 
     ```python
     >>> model = ModelWithLoraLayer()
-    >>> alpha = 0.5
-    >>> with set_adapter_scale(model, alpha):
+    >>> multiplier = 0.5
+    >>> with rescale_adapter_scale(model, multiplier):
     ...     outputs = model(**inputs)  # Perform operations with the scaled model
     >>> outputs = model(**inputs)  # The original scaling values are restored here
     ```
     """
-    # check if alpha has a valid data type
-    if not isinstance(alpha, (float, int)):
-        raise TypeError(f"{alpha} should be of type float, got {type(alpha)}")
+    # check if multiplier has a valid data type
+    if not isinstance(multiplier, (float, int)):
+        raise TypeError(f"{multiplier} should be of type float, got {type(multiplier)}")
```

Comment:
Oh, just a small issue I missed originally: I think it's better to make the message: `"Argument multiplier should be..."` and not use the value itself.